ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -1088,7 +1087,7 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
-- as: azure-ipi-sno-arm-f28
+- as: azure-ipi-sno-etcd-encryption-arm-f28
```

Comment:
hmm, we are going to update other platforms in this PR?

Reply:
Yes, cover more platforms.

Reply:
For aws, you added a dedicated install workflow for sno + etcd-encrypt, so to be consistent, I suggest to add dedicated install workflow for other platforms.

Reply:
Sure.